.gitignore

```diff
@@ -33,6 +33,9 @@ qa/.vm_ssh_config
 qa/.vagrant
 qa/acceptance/.vagrant
 qa/Gemfile.lock
+Gemfile.jruby-1.9.lock
+.lock
```

Comment:
Weird, I haven't seen that one before, lets remove it. so when it happen again we can find it.